recipes/nd2reader/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "nd2reader" %}
+{% set version = "3.0.0" %}
+{% set sha256 = "18c721250669de804b85b7295e9885c94c21f99918b8da451ebbbab84fe29612" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - numpy >=1.6.2
```

Comment:
Sorry for adding another request, but I unfortunately forgot it in my initial review. When you use `--single-version-externally-managed` you don't need to specify what is included in `install_requires` in the `setup.py`, unless `setup.py` actually attempts to import them or something inside the package that does. So I think you only need `python` and `setuptools` in the build section.